templates/zerver/integrations/big-blue-button.md

```diff
@@ -1,35 +1,32 @@
-Zulip supports using BigBlueButton as its video call video call
-provider. This is currently only possible on self-hosted Zulip
-installations.
+# Zulip BigBlueButton integration
 
-To use the [BigBlueButton](https://bigbluebutton.org/) video call
-integration, you'll need to have a BigBlueButton server and
-configure your zulip server to use that BigBlueButton server.
+Zulip supports using BigBlueButton as its video call provider.
 
-### Configure BigBlueButton server
```

Comment:
Merged all the different sections into a single tabbed section, as 2 sections could be written in just 1 step each.